nz.mega.MEGAsync.yml

```diff
@@ -0,0 +1,166 @@
+id: nz.mega.MEGAsync
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+runtime-version: '5.15'
+finish-args:
+  - --filesystem=~/MEGA:create
```

Comment:
Would be nice if this could be avoided. creating files & folders directly under ~ is something that some users do not like.  In my system ~ is read-only so this gonna fail.  
I believe also changing that location via the app's settings will fail due to the fact `DontUseNativeDialog` is set as false (`src/MEGASync/{gui/MultiQFileDialog,MegaApplication}.cpp`) and selecting folders via XDG File portal does not work correctly (or at least with this app).
Can this location be patched in the app to default to another location? like `xdg-documents/MEGA` or `xdg-public-share/MEGA`

Reply:
> creating files & folders directly under ~ is something that some users do not like.

Not working app out of box is that something that most users do not like even more.

> Can this location be patched in the app to default to another location? like xdg-documents/MEGA or xdg-public-share/MEGA

Both locations looks like not proper place for this app and also insecure.

I would stick with upstream defaults.

Reply:
But for `DontUseNativeDialog` - i would try to experiment locally with this as soon as i can. Would be nice to make portals works for sure. Thanks for tip.

Reply:
@tinywrkb you right, platform-native file dialog doesn't work correctly and would be really nice if @grulja help us at least to point where the trouble - in portals/runtime or this is 'MEGA' bug?

I've also found your treasure [here](https://github.com/tinywrkb/flatpaks/blob/master/nz.mega.megasync/nz.mega.megasync.yaml) so i've enabled for debugging purposes platform-native file dialog in `2d9dcdf` for now.

Reply:
@tim77 I should have limited my changes to affect only the folder selection setting as we still want to be able to use the XDG File portal file selection for upload/download.
Also, thanks for the attribution

Reply:
We will able to use **native** XDG File portal if we will fix it entirely (in app itself, probaly). :) This will be win-win solution. You able to use file selection now only because of `--filesystem=home` in your build which if far from ideal.